engine/src/main/java/org/hibernate/search/engine/metadata/impl/AnnotationMetadataProvider.java

```diff
@@ -1040,7 +1040,9 @@ private void initializeMemberLevelAnnotations(String prefix,
 		);
 		checkForContainedIn( member, typeMetadataBuilder, parseContext );
 
-		numericFields.validate();
+		if ( !parseContext.skipFieldBridges() ) {
```

Comment:
It's definitely not the only puzzling thing in this part of the code :D But yes, you're right.  I'll add a comment :)